pkgs/development/python-modules/lime/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+
+, matplotlib, numpy, scipy, tqdm, scikit-learn-extra, scikit-image
+
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "lime";
+  version = "0.2.0.1";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-dpYOTwVf61Pom1AiODuvyHtj8lusYmWYSwozPRpX94E=";
+  };
+
+  propagatedBuildInputs =  [
+    matplotlib
+    numpy
+    scipy
+    tqdm
+    scikit-learn-extra
+    scikit-image
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  disabledTestPaths = [
+    # touches network
+    "lime/tests/test_lime_text.py"
+  ];
+
```

Comment:
Would you please add `pythonImportsCheck`?

Reply:
Of course, I have added modules listed in [module doc's](https://github.com/marcotcr/lime/blob/master/doc/lime.rst) to `pythonImportsCheck`. Thanks for fast review.